lib/BackgroundJob/RepairRecipients.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Mail\BackgroundJob;
+
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\IJobList;
+use OCP\BackgroundJob\TimedJob;
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use OCP\IDBConnection;
+
+class RepairRecipients extends TimedJob {
+
+	public function __construct(
+		protected ITimeFactory $time,
+		private IDBConnection $db,
+		private IJobList $jobService,
+	) {
+		parent::__construct($time);
+		$this->setInterval(300);
+	}
+
+	protected function run($argument): void {
+		// fetch all quoted emails
+		$select = $this->db->getQueryBuilder();
+		$select->select('id', 'email')
+			->from('mail_recipients')
+			->where(
+				$select->expr()->like('email', $select->createNamedParameter('\'%\'', IQueryBuilder::PARAM_STR))
+			)
+			->setMaxResults(1000);
+		$recipients = $select->executeQuery()->fetchAll();
+		// update emails
+		$update = $this->db->getQueryBuilder();
+		$update->update('mail_recipients')
+			->set('email', $update->createParameter('email'))
+			->where($update->expr()->in('id', $update->createParameter('id'), IQueryBuilder::PARAM_STR));
+		foreach ($recipients as $recipient) {
+			$id = $recipient['id'];
+			$email = $recipient['email'];
+			$email = trim(str_replace('\'', '', (string)$email));
```

Comment:
```suggestion
			$email = trim(trim($email, "'"));
```

The best approach would be to reuse the logic from the address class. If we don't go with the helper/utility, we should still use trim to remove the first and last character (even if we can be very sure, due to the query, that we're only dealing with emails starting and ending with a `'`).